test/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+###############
+# Build tests
+###############
+
+################
+# Add tests to ctest
+################
+
+ADD_EXECUTABLE(shuffile_test test1.c)
+TARGET_LINK_LIBRARIES(shuffile_test ${SHUFFILE_EXTERNAL_LIBS} shuffile)
+ADD_TEST(NAME shuffile_test COMMAND ${MPIEXEC} -n6 -N3 -m block ./shuffile_test)
```

Comment:
@kosinovsky, Travis CI is set up, but test fails with current hard coded options.
```
1: Test command: /usr/bin/mpiexec "-n6" "-N3" "-m" "block" "./shuffile_test"
1: Test timeout computed to be: 9.99988e+06
1: --------------------------------------------------------------------------
1: mpiexec was unable to launch the specified application as it could not find an executable:
1: 
1: Executable: -6
1: Node: travis-job-924d1308-0162-4ecf-acd7-731a5affb540
1: 
1: while attempting to start process rank 0.
1: --------------------------------------------------------------------------
1/1 Test #1: shuffile_test ....................***Failed    0.01 sec
```

Reply:
I hardcoded number of processes and number of nodes because I didn't know the 
`${MPIEXEC_NUMPROC_FLAG}` -- equivalent for the number of nodes. This needs to be fixed. @gonsie -- what is the equivalent for the number of nodes?

Reply:
There is no num-nodes equivalent. You should probably leave it hard coded for now.

Reply:
Should we just not run `make check` in travis for now?